README.md

```diff
@@ -25,6 +25,19 @@ This repository specifically contains these PDS registry application resources:
 All users and developers of the NASA-PDS software are expected to abide by our [Code of Conduct](https://github.com/NASA-PDS/.github/blob/main/CODE_OF_CONDUCT.md). Please read this to ensure you understand the expectations of our community.
 
 
+## Utilities
+* Titan Treks
+    * The Titan Treks utilities can be used to create pds4 labels for the layers given in the Titan Treks API
+    * From the `utils/titan_treks/` directory, run `create_titan_treks_pds4.py`:
```

Comment:
Thanks to the configuration in the `setup.cfg` you should be able to call the command line like that `create-titan-treks-pds4` without need to go to the directory where your script is. And then you would not even need to have the detailed parameter in the readme but you can just ask user to run `create-titan-treks-pds4 --help`.
Since you added the detailed parameters you can keep them now, that does not hurt.

Reply:
This ☝️